sympy/physics/optics/gaussopt.py

```diff
@@ -582,9 +588,10 @@ def radius(self):
         return self.z*(1 + (self.z_r/self.z)**2)
 
     @property
-    def w(self):
+    def w_z(self):
```

Comment:
PR is not letting me unresolve conversation so I am adding comment here:

I don't find the word "spot" in SymPy -- where is the spot size talked about? It is not yet clear why this should be changed from `w` to `w_z`.

Reply:
I'll unresolve that conversation ,thanks !

Reply:
Unresolved and addressed that change !